crates/sncast/tests/e2e/account/create.rs

```diff
@@ -185,12 +185,11 @@ pub async fn test_happy_case_add_profile() {
         "my_account",
     ];
 
-    let snapbox = runner(&args).current_dir(tempdir.path());
-    let output = snapbox.assert().success();
+    let output = runner(&args).current_dir(tempdir.path()).assert();
 
     assert_stdout_contains(
         output,
-        "add_profile: Profile my_account successfully added to snfoundry.toml",
+        "add_profile: Profile my_account successfully added to [..]",
```

Comment:
Can't we use the tempdir path joined with something to assert proper paths in these tests?